### PR TITLE
Add back "seldon" gateway

### PR DESCRIPTION
**Pull request by adriangonz**
**What this PR does / why we need it**:

Adds back the `seldon` gateway (which was removed in #2623), but removing the OAuth-specific code. 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
```

### CODE REVIEW

**Comment by seldondev** [comment]
Tue Nov 24 11:32:40 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2701/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2701 --build=2

**Comment by seldondev** [comment]
Tue Nov 24 11:32:41 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2701/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2701 --build=1

**Comment by seldondev** [comment]
Tue Nov 24 11:34:44 UTC 2020
The logs for [pr-build] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2701/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2701 --build=3

**Comment by seldondev** [comment]
Tue Nov 24 11:34:41 UTC 2020
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2701/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2701 --build=4

File: python/seldon_core/seldon_client.py [hunk 1]
```diff
@@ -1717,6 +1911,135 @@ def grpc_predict_gateway(
         return SeldonClientPrediction(request, None, False, str(e))
 
 
+def rest_feedback_seldon(
```
Comment (RafalSkolasinski) [inline python/seldon_core/seldon_client.py:1914]: We may need to include `raw_request` like in `rest_feedback_gateway`

File: python/seldon_core/seldon_client.py [hunk 2]
```diff
@@ -1717,6 +1911,135 @@ def grpc_predict_gateway(
...
+            except:
+                response = None
+        else:
+            response = None
+        return SeldonClientFeedback(request, response, success, msg)
+    except Exception as e:
+        return SeldonClientFeedback(request, None, False, str(e))
+
+
+def grpc_feedback_seldon(
```
Comment (RafalSkolasinski) [inline python/seldon_core/seldon_client.py:1977]: we need to include `raw_request` like in `grpc_feedback_gateway`

**Review by RafalSkolasinski — CHANGES_REQUESTED** [review]
Nice one, that was fast. 
I added comments about including `raw_request` for feedback -> required for batch processing

File: python/seldon_core/seldon_client.py (hunk 2 above)
Comment (adriangonz) [reply python/seldon_core/seldon_client.py:1977]: Got it, let me add that

**Comment by seldondev** [comment]
Tue Nov 24 12:02:09 UTC 2020
The logs for [lint] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2701/6.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2701 --build=6

**Comment by seldondev** [comment]
Tue Nov 24 12:02:47 UTC 2020
The logs for [pr-build] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2701/5.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2701 --build=5

**Comment by seldondev** [comment]
Tue Nov 24 12:14:44 UTC 2020
The logs for [pr-build] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2701/7.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2701 --build=7

**Comment by seldondev** [comment]
Tue Nov 24 14:15:45 UTC 2020
The logs for [pr-build] [8] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2701/8.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2701 --build=8

**Comment by seldondev** [comment]
Tue Nov 24 14:16:03 UTC 2020
The logs for [lint] [9] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2701/9.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2701 --build=9

**Comment by seldondev** [comment]
Tue Nov 24 14:40:24 UTC 2020
The logs for [lint] [11] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2701/11.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2701 --build=11

**Comment by seldondev** [comment]
Tue Nov 24 14:41:32 UTC 2020
The logs for [pr-build] [10] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2701/10.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2701 --build=10

**Comment by seldondev** [comment]
@adriangonz: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 9720d596a8563d0d2bc6a8f9bdb1341e1bcc104a | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Pull request merged**